framework/components/ecl-datepickers/_datepickers.scss

```diff
@@ -40,7 +40,7 @@ $ecl-datepickers-week-color: #999 !default;
   color: $ecl-datepickers-text-color;
   display: block;
   position: relative;
-  z-index: 9999;
+  z-index: map-get($ecl-z-index, 'modal');
```

Comment:
To me it is more like a popup (= modal).
It has to be on top of other elements (including dropdowns)